# Request
---
# No request fields needed for this service

# Response
bool success
